src/@utils/schaclSchema.ts

```diff
@@ -0,0 +1,42 @@
+import { retrieveShaclSchema } from '@utils/aquarius'
+import { ShaclSchemaField } from '@context/MarketMetadata/_shaclType'
+
+import { capitalizeFirstLetter } from '@utils/textTransform'
+
+function getMinMax(valueField: string, field: ShaclSchemaField) {
+  const min = field.minLength
+  const max = field.maxLength
+
+  return {
+    minLength: min,
+    maxLength: max
+  }
+}
+
+export async function validateFieldSchaclSchema(
+  path: string,
+  value: string,
+  createError: any
```

Comment:
this `createError` should be typed as its always same usage so pretty sure we can figure out its typing